rust/arrow/src/compute/kernels/sort.rs

```diff
@@ -881,6 +913,33 @@ mod tests {
         assert_eq!(output, expected)
     }
 
+    // TODO remove this function
```

Comment:
I will rewrite the tests properly (with a single `SortOptions` parameter) if people like this API change, but I didn't bother until I know it is worthwhile